From ba48a2636858545aa2ca6d152c9f437c5dd538a6 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Thu, 27 Dec 2012 18:55:16 +0800
Subject: [PATCH 132/310] ENGR00238396 imx6: enable the quick settings panel
 on status bar

The Settings.Secure.USER_SETUP_COMPLETE must be set
to true, otherwise the quick settings panel would
be disabled for status bar. Meanwhile, if we
want to add camera and search shortcut for the
Keyguard unlock pad, this secure field should
also be set to true.

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 .../SettingsProvider/res/values/defaults.xml       | 22 ++++++++++++++++++++++
 .../SettingsProvider/res/values/defaults.xml       | 22 ++++++++++++++++++++++
 2 files changed, 44 insertions(+)
 create mode 100644 sabreauto_6q/overlay/frameworks/base/packages/SettingsProvider/res/values/defaults.xml
 create mode 100644 sabresd_6dq/overlay/frameworks/base/packages/SettingsProvider/res/values/defaults.xml

diff --git a/sabreauto_6q/overlay/frameworks/base/packages/SettingsProvider/res/values/defaults.xml b/sabreauto_6q/overlay/frameworks/base/packages/SettingsProvider/res/values/defaults.xml
new file mode 100644
index 0000000..3f2232a
--- /dev/null
+++ b/sabreauto_6q/overlay/frameworks/base/packages/SettingsProvider/res/values/defaults.xml
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+/**
+ * Copyright (c) 2009, The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+-->
+<resources>
+    <!-- Default for Settings.Secure.USER_SETUP_COMPLETE -->
+    <bool name="def_user_setup_complete">true</bool>
+</resources>
diff --git a/sabresd_6dq/overlay/frameworks/base/packages/SettingsProvider/res/values/defaults.xml b/sabresd_6dq/overlay/frameworks/base/packages/SettingsProvider/res/values/defaults.xml
new file mode 100644
index 0000000..3f2232a
--- /dev/null
+++ b/sabresd_6dq/overlay/frameworks/base/packages/SettingsProvider/res/values/defaults.xml
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+/**
+ * Copyright (c) 2009, The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+-->
+<resources>
+    <!-- Default for Settings.Secure.USER_SETUP_COMPLETE -->
+    <bool name="def_user_setup_complete">true</bool>
+</resources>
-- 
1.8.0

